linux/vtpm_cold_add_remove/clear_tpm_test.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Check OS still work well after clear TPM
+#
+- name: "Clear vTPM in guest OS"
+  ansible.builtin.shell: tpm2_clear
+  delegate_to: "{{ vm_guest_ip }}"
+
+- name: "Restart guest OS after reset TPM"
```

Comment:
Need to use the task file in utils to restart guest OS since VMware Tools can be not available.